portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,14 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Joey Bushagour's Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>I'm Joey, a STEP intern at Google and a rising sophomore at Purdue University. In my free time, I like gardening, hardware projects, and going on walks. </p>
```

Comment:
Nit: There's not a hard limit on line length for HTML, but 80 or 100 are good options. It makes the code more readable and easier to review without scrolling.

Reply:
Thank you! Updated with fix.